FAIL
Kernel panic in run: PANIC at ../../threads/synch.c:265 in lock_release(): assertion `lock_held_by_current_thread (lock)' failed.
Call stack: 0x800421894c 0x800420ae40 0x800421dfc9 0x800421da1e 0x800421d771 0x4002e6 0x40035a 0x400196 0x400c1c
Translation of call stack:
0x000000800421894c: debug_panic (lib/kernel/debug.c:32)
0x000000800420ae40: lock_release (threads/synch.c:268)
0x000000800421dfc9: write (userprog/syscall.c:406)
0x000000800421da1e: syscall_handler (userprog/syscall.c:168)
0x000000800421d771: no_sti (userprog/syscall-entry.o:?)
0x00000000004002e6: (unknown)
0x000000000040035a: (unknown)
0x0000000000400196: (unknown)
0x0000000000400c1c: (unknown)
